File: Coding/Python/fibonacci.py
# Script to output the first n numbers of the fibonacci number sequence
#!/usr/bin/python
def fibonacci(num):
    i = 0
    fibonacci = [0, 1]
    while i < num-1:
        fibonacci.append(fibonacci[i]+fibonacci[i+1])
        i+=1
    return fibonacci

num = 100
list = fibonacci(num)

print list

File: Coding/Perl/fibonacci.pl
# Script to output the first n numbers of the fibonacci number sequence
#!/usr/bin/perl
use strict;
use warnings;
use diagnostics;
use 5.010;

my @list = ();
my $times = 100;

@list = &fibonacci($times);

print "@list\n";

sub fibonacci(){
    my $num=$_[0];
    my $i=0;
    my @list=(0,1);
    
    while($i<($num-1)){
        push @list, $list[$i]+$list[$i+1];
        $i++;
    } 
    return @list;
}